src/templates/admin/submission/submit_files.html

```diff
@@ -20,19 +20,17 @@
         <div class="row expanded">
             <div class="large-12 columns">
 
-                <p><strong>{% trans "Submission guidelines" %}</strong>: {% blocktrans %}To upload a file, select it
-                    using one of the 'Choose file' buttons, then upload it with the 'Upload file' button next to it. You
-                    will then be asked for some additional information (label, description etc.){% endblocktrans %}</p>
+                <p>{{ file_submission_guidelines }}</p>
```

Comment:
We loose the "**Submission Guidelines**" bold text here, which is not necessarily a problem

Reply:
Yes, that is intentional. I wanted to give editors control over all the text that appears here.

Reply:
makes sense